code/modules/surgery/healing.dm

```diff
@@ -6,14 +6,23 @@
 				/datum/surgery_step/heal,
 				/datum/surgery_step/close)
 
-	target_mobtypes = list(/mob/living/carbon/human, /mob/living/carbon/monkey)
+	target_mobtypes = list(/mob/living)
 	possible_locs = list(BODY_ZONE_CHEST)
 	requires_bodypart_type = FALSE
 	replaced_by = /datum/surgery
 	ignore_clothes = TRUE
 	var/healing_step_type
 	var/antispam = FALSE
 
+/datum/surgery/healing/can_start(mob/user, mob/living/patient)
+	. = ..()
+	if(isanimal(patient))
+		var/mob/living/simple_animal/critter = patient
+		if(!critter.healable)
+			return FALSE
+	if(!(patient.mob_biotypes & MOB_ORGANIC))
```

Comment:
so, uh, is it intentional that you're making the only ways to heal skeletons bolts of death, nanites, legion cores, and milk (they can't process most healing chems, can't be sutured (last I checked), and with this PR, won't be able to benefit from tend wounds)? 'cause the away mission skeleton has access to none of those things (bar legion cores), and usually had to have his golem minions tend wounds him up.

Reply:
also, this means that plasmamen are now FUCKED, because they can't be tend wounds'd up into defib range- you HAVE to use synthflesh (or a bolt of healing) on them to bring them into defib range now

Reply:
please, please, PLEASE remove the biotype restriction here, it's gonna mess so much up

Reply:
i'm pretty sure nanites need the organic biotype to work as well

Reply:
> i'm pretty sure nanites need the organic biotype to work as well

they need either the organic biotype or the undead biotype

Reply:
i tested weird species by running android which had organic so i thought all species carried it, ive added humanoid